src/Aks/Aks/Commands/NewAzureRmAks.cs

```diff
@@ -34,6 +34,7 @@
 
 namespace Microsoft.Azure.Commands.Aks
 {
+    [GenericBreakingChangeWithVersion("The default created service principal will only have contributor role for this resource group","11.1.0","6.0.0")]
```

Comment:
```suggestion
    [GenericBreakingChangeWithVersion("The default created service principal will only have contributor role for this resource group","12.0.0","7.0.0")]
```
The target version of Az should be 12.0.0 and Az.Aks should be 7.0.0